vars/gruntBuild.groovy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/groovy
+
+def call(body) {
+    def config = [:]
```

Comment:
Also, a lot of inspiration is taken from https://github.com/fabric8io/fabric8-pipeline-library. You might want to checkout some of there examples.